### PR TITLE
Drop use of test() in templates 

### PR DESCRIPTION
**Pull request by pbauer**
It is no longer available since Zope 4.5.1 (see https://community.plone.org/t/plone-5-2-1-5-2-2-page-template-issues/12793)

### CODE REVIEW

File: Products/ATContentTypes/skins/ATContentTypes/atct_topic_pdf.pt [hunk 1]
```diff
@@ -38,7 +38,7 @@
                                        item_created item/CreationDate;
                                        item_wf_state        item/review_state;
                                        item_creators item/Creators|item/Creator;
-                                       item_creators python:test(same_type(item_creators,[]) or same_type(item_creators,()), item_creators, [item_creators]);
+                                       item_creators python:item_creators if same_type(item_creators,[]) or same_type(item_creators,()) or [item_creators];
```
Comment (mauritsvanrees) [inline Products/ATContentTypes/skins/ATContentTypes/atct_topic_pdf.pt:41]: Typo. The last `or` should be `end`.

**Review by mauritsvanrees — CHANGES_REQUESTED** [review]
One wrong replacement found, see inline comment.
Looks good for the rest. Thanks!
Please add a changelog entry.
And start the Jenkins tests. I don't suppose these files are really tested, otherwise this would have been caught already. But maybe a typo that breaks the html structure can be found.

**Comment by pbauer** [comment]
@jenkins-plone-org please run jobs

File: Products/ATContentTypes/skins/ATContentTypes/atct_topic_pdf.pt (hunk 1 above)
Comment (pbauer) [reply Products/ATContentTypes/skins/ATContentTypes/atct_topic_pdf.pt]: done

**Review by mauritsvanrees — APPROVED** [review]

**Pull request merged**